srml/council/src/lib.rs

```diff
@@ -159,11 +163,11 @@ mod tests {
 		type EmergencyVotingPeriod = VotingPeriod;
 		type VotingPeriod = VotingPeriod;
 		type MinimumDeposit = MinimumDeposit;
-		type ExternalOrigin = motions::EnsureProportionAtLeast<_1, _2, u64>;
-		type ExternalMajorityOrigin = motions::EnsureProportionAtLeast<_2, _3, u64>;
-		type EmergencyOrigin = motions::EnsureProportionAtLeast<_1, _1, u64>;
-		type CancellationOrigin = motions::EnsureProportionAtLeast<_2, _3, u64>;
-		type VetoOrigin = motions::EnsureMember<u64>;
+		type ExternalOrigin = motions::EnsureProportionAtLeast<_1, _2, u64, &'static str>;
```

Comment:
do we really have to have the extra generic param here?

Reply:
I don't know why this parameter was there, it is now removed.